tests/Resources/Page/ODH/Prometheus/Prometheus.robot

```diff
@@ -6,16 +6,29 @@ Library             Process
 Library             RequestsLibrary
 
 *** Keywords ***
+Get Sep Token
+    [Documentation]  Returns the token to access observatorium
+    ${data}=    Create Dictionary     grant_type=client_credentials  client_id=${STARBURST.OBS_CLIENT_ID}  client_secret=${STARBURST.OBS_CLIENT_SECRET}
+    ${resp}=     RequestsLibrary.POST  ${STARBURST.OBS_TOKEN_URL}   data=${data}
+    [Return]    ${resp.json()['access_token']}
+
 Run Query
     [Documentation]    Runs a prometheus query, obtaining the current value. More info at:
     ...                - https://promlabs.com/blog/2020/06/18/the-anatomy-of-a-promql-query
     ...                - https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries
-    [Arguments]    ${pm_url}    ${pm_token}    ${pm_query}
+    [Arguments]    ${pm_url}    ${pm_token}    ${pm_query}  ${project}=RHODS
     ${pm_headers}=    Create Dictionary    Authorization=Bearer ${pm_token}
-    ${resp}=    RequestsLibrary.GET    url=${pm_url}/api/v1/query?query=${pm_query}
-    ...    headers=${pm_headers}    verify=${False}
-    Status Should Be    200    ${resp}
-    # Log To Console    ${resp.json()}
+
+    IF  "${project}" == "SEP"
```

Comment:
can you call it SERH (i.e. starburst enterprise for red hat or Managed Starburst)?

Reply:
sure